test_scripts/API/Navigation/UnsubscribeWayPoints/010_UnsubscribeWayPoints_Transfering_of_HMI_error_resultCode_to_mobile.lua

```diff
@@ -0,0 +1,158 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/27
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Unsubscribe_from_Destination_and_Waypoints.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [UnsubscribeWayPoints] As a mobile app I want to be able to unsubscribes from getting notifications
+-- about any changes to the destination or waypoints.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sent valid and allowed by Policies UnsubscribeWayPoints_request to SDL
+--
+-- SDL must:
+-- 1) transfer UnsubscribeWayPoints_request_ to HMI
+-- 2) respond with <resultCode> received from HMI to mobile app
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonNavigation = require('test_scripts/API/Navigation/commonNavigation')
+
+--[[ Local Variables ]]
+local successResultCodes = {
+  "SUCCESS",
+}
+
+local failureResultCodes = {
+  "INVALID_DATA",
+  "GENERIC_ERROR",
+  "REJECTED",
+  "UNSUPPORTED_RESOURCE",
+  "IGNORED",
+  "DISALLOWED"
+}
+
+local unexpectedResultCodes = {
```

Comment:
@dboltovskyi I propose not to hard code these codes but read needed from API.xml.
Any LUA parser can be used. This will be useful for all further code writers. 
[LuaXML](http://lua-users.org/wiki/LuaXml)